src/lib/models/types/reference.ts

```diff
@@ -115,10 +115,15 @@ export class ReferenceType extends Type
      * Return a string representation of this type.
      */
```

Comment:
You're right. I was also looking for some test for the class but I didn't it. Is there any? Thanks for your review (: